changelogs/current.yaml

```diff
@@ -434,6 +434,13 @@ new_features:
   change: |
     Added :ref:`routeName() <config_http_filters_lua_stream_info_route_name>` API to the Stream Info Object to get the
     name of the route matched by the filter chain.
+- area: oauth2
+  change: |
+    Add the option to specify samesite cookie attribute value for oauth2 supported cookies.
```

Comment:
nit: s/samesite/SameSite and s/oauth/OAuth
here and below

Reply:
SameSite adjusted. oauth2 remains same as it is all lower-cased in previous change logs